source/Calamari.AzureCloudService/Calamari.AzureCloudService.csproj

```diff
@@ -26,6 +24,8 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Calamari.AzureScripting\Calamari.AzureScripting.csproj" />
+    <ProjectReference Include="..\Calamari.Common\Calamari.Common.csproj" />
+    <ProjectReference Include="..\Calamari.Scripting\Calamari.Scripting.csproj" />
```

Comment:
Not sure if these are needed, I think they should have been transitively included from `Calamari.AzureScripting` now. But I'm happy either way